subprojects/logging-api/src/main/java/org/gradle/api/logging/configuration/ConsoleOutput.java

```diff
@@ -24,6 +28,13 @@ public enum ConsoleOutput {
      * Disable all color and rich output. Generate plain text only.
      */
     Plain,
+    /**
+     * Disable all color and rich output. Generate plain text only. Occasionally reports progress.
+     *
+     * @since 8.4
```

Comment:
This won't make it into Gradle `8.4`; please change this to `8.5` (and others, if there are any).